doc/定義/フロント/画面/テンプレート.md

```
@@ -0,0 +1,78 @@
+この画面で使うapi
+
+例:
+メモ取得api
+
+# レスポンス受取後
+
+レスポンスを受け取った後の処理
+
+いちいち言わんくてもわかるようなことは書くだけ疲れるのでかかない。
+
+### エラー時
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)